### PR TITLE
chore(short-survey): add web compat survey

### LINKED ISSUE #8087: chore: run "Help prioritise cross-browser web platform features" survey
The Firefox team wants to run a short survey to determine what the most important web compat issue is to work on.

> Ensuring the web is open and accessible to all is central to Mozilla’s mission. The Firefox product team is interested in learning which features are most important to support across all major browsers.

Sampling rate: 5% of JavaScript pages
Run for 7 days

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds the webcompat survey. This should run until Monday (2 - 6 February) for 10% of the audience. We might bump it to next week Wednesday if there have not been enough responses.

fix #8087

## Screenshots

image

image

### CODE REVIEW

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 1]
```diff
@@ -20,18 +20,32 @@ export interface Survey {
...
   INTEROP_2023_API_JS = "INTEROP_2023_API_JS",
 }
 
 export const SURVEYS: Survey[] = [
+  {
+    key: SurveyKey.FIREFOX_WEB_COMPAT,
+    bucket: SurveyBucket.FIREFOX_WEB_COMPAT,
+    show: (doc: Doc) =>
+      /en-US\/docs\/Web\/(JavaScript|API)(\/|$)/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/7195211/Which-web-platform-features-should-be-implemented-in-Firefox",
```
Comment (zcorpan) [inline client/src/ui/molecules/document-survey/surveys.ts:42]: Hmm can we update the URL to match the new title?

Comment (zcorpan) [reply client/src/ui/molecules/document-survey/surveys.ts:42]: I've edited the URL in Alchemer.
```suggestion
    src: "https://survey.alchemer.com/s3/7195211/Help-prioritize-cross-browser-web-platform-features",
```

**Review by zcorpan — CHANGES_REQUESTED** [review]

**Review by zcorpan — APPROVED** [review]

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 2]
```diff
@@ -20,18 +20,32 @@ export interface Survey {
...
   CSS_CASCADE_2022 = "CSS_CASCADE_2022",
+  FIREFOX_WEB_COMPAT = "FIREFOX_WEB_COMPAT",
   INTEROP_2023 = "INTEROP_2023",
 }
 
 enum SurveyKey {
   BROWSER_SURVEY_OCT_2022 = "BROWSER_SURVEY_OCT_2022",
   CSS_CASCADE_2022_A = "CSS_CASCADE_2022_A",
   CSS_CASCADE_2022_B = "CSS_CASCADE_2022_B",
+  FIREFOX_WEB_COMPAT = "FIREFOX_WEB_COMPAT",
```
Comment (caugner) [inline client/src/ui/molecules/document-survey/surveys.ts:31]: dito

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 3]
```diff
@@ -20,18 +20,32 @@ export interface Survey {
...
   INTEROP_2023_CSS_HTML = "INTEROP_2023_CSS_HTML",
   INTEROP_2023_API_JS = "INTEROP_2023_API_JS",
 }
 
 export const SURVEYS: Survey[] = [
+  {
+    key: SurveyKey.FIREFOX_WEB_COMPAT,
+    bucket: SurveyBucket.FIREFOX_WEB_COMPAT,
```
Comment (caugner) [inline client/src/ui/molecules/document-survey/surveys.ts:39]: dito

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 4]
```diff
@@ -20,18 +20,32 @@ export interface Survey {
...
+    show: (doc: Doc) =>
+      /en-US\/docs\/Web\/(JavaScript|API)(\/|$)/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/7195211/Help-prioritize-cross-browser-web-platform-features",
+    teaser:
+      "Ensuring the web is open and accessible to all is central to Mozilla’s mission. The Firefox product team is interested in learning which features are most important to support across all major browsers.",
+    question: "Let us know what you think.",
+    ...survey_duration(SurveyBucket.FIREFOX_WEB_COMPAT),
+    ...survey_rates(SurveyKey.FIREFOX_WEB_COMPAT),
```
Comment (caugner) [inline client/src/ui/molecules/document-survey/surveys.ts:47]: dito

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, let's just add a year to the enum/key.

**Review by caugner — APPROVED** [review]
LGTM, tested locally.

**Pull request merged**